### DISCUSSION TITLE
Sort by state

### DISCUSSION THREAD

**Discussion by agneevX** [jesec/flood#257] [Ideas]
Hi!

Would be nice if we could sort by state. 

E.g. the torrents on top would be the ones that are currently active.

**Comment by jesec** [comment]
Simply sort by download speed or upload speed. a torrent is active when it has upload or download speed.

**Reply by agneevX** [reply]
Sure, but what if you want to track both at the same time. Just felt like a feature request since its on every qBittorrent frontend I've used and is super useful to me.
